docs/user/secure-gateways.md

```diff
@@ -0,0 +1,214 @@
+# Secure Gateways
+
+This guide will help you get started using secure Gateways. The guide uses a self-signed CA, so it should be used for
+testing and demonstration purposes only.
+
+## Prerequisites
+
+- A Kubernetes cluster with `kubectl` context configured for the cluster.
+- OpenSSL to generate TLS assets.
+
+__Note:__ Envoy Gateway is tested against Kubernetes v1.24.
+
+## Installation
+
+Follow the steps from the [Quickstart Guide](QUICKSTART.md) to install Envoy Gateway and the example manifest.
+Before proceeding, you should be able to curl the example backend using HTTP.
+
+## TLS Certificates
+
+Generate the certificates and keys used by the Gateway to terminate client TLS connections.
+
+For macOS users, verify curl is compiled with the LibreSSL library:
```

Comment:
Why does that matter?

Reply:
I referenced this from https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/#before-you-begin. I'll remove.